sdk/core/azure-core/azure/core/pipeline/policies/_universal.py

```diff
@@ -222,7 +226,7 @@ def on_request(self, request):
             if options_dict.pop('user_agent_overwrite', self.overwrite):
                 http_request.headers[self._USERAGENT] = user_agent
             else:
-                user_agent = "{} {}".format(self.user_agent, user_agent)
+                user_agent = "{} {}".format(user_agent, self.user_agent)
```

Comment:
Is the reason for switching the order documented anywhere?

Reply:
From Johan:

See https://azure.github.io/azure-sdk/general_azurecore.html#telemetry-policy:

[<application_id> ]azsdk-<sdk_language>-<package_name>/<package_version> <platform_info>

I would expect the caller to provide the application id as a keyword argument, and I would expect the library to provide the sdk-language/package-name/package-version and azure-core to provide the platform info.

Reply:
From Johan:

See https://azure.github.io/azure-sdk/general_azurecore.html#telemetry-policy:

[<application_id> ]azsdk-<sdk_language>-<package_name>/<package_version> <platform_info>

I would expect the caller to provide the application id as a keyword argument, and I would expect the library to provide the sdk-language/package-name/package-version and azure-core to provide the platform info.

Reply:
This is the sort of information it is good (IMO) to record in associated issues.